rust-runtime/aws-smithy-observability/README.md

```diff
@@ -0,0 +1,7 @@
+# aws-smithy-observability
+
+This crate contains traits allowing for the implementation of `TelemetryProvider`s for the AWS SDK for Rust. I also contains a `global` module for setting and interacting with the current `GlobalTelemetryProvider`.
```

Comment:
I -> It also contains

Reply:
Fixed in https://github.com/smithy-lang/smithy-rs/pull/3912/commits/0559e8b54fb41bcf04125e2e3cedeec87b24a9f9